src/templates/en/2019/base.html

```diff
@@ -64,7 +64,7 @@
 {% block written_by_before %}Written by{% endblock %}
 {% block reviewed_by_before %}Reviewed by{% endblock %}
 {% block translated_by_before %}Translated by{% endblock %}
-{% block analysis_by_before %}Analysis by{% endblock %}
+{% block analysis_by_before %}Analysed by{% endblock %}
```

Comment:
en-US ;)
```suggestion
{% block analysis_by_before %}Analyzed by{% endblock %}
```

Reply:
I remember thinking when I started on this branch to check that! Then obviously completely forgot 😔